web/ui/package.json

```diff
@@ -33,5 +36,10 @@
     "merge-stream": "1.0.0",
     "run-sequence": "1.2.2",
     "which": "1.2.11"
+  },
+  "dependencies": {
+    "angular-auth0": "^3.0.0",
+    "auth0": "^2.9.1",
+    "auth0-js": "^9.4.2"
```

Comment:
It would normally be correct to add these to `dependencies`, but we are effectively vendoring all of our deps, so they should not appear in this list.